.travis.yml

```diff
@@ -32,7 +32,9 @@ install:
   - npm install && npm run lint
 
 script:
-  - npm run build_spec && npm run test_mocha && node ./node_modules/markdown-doctest/bin/cmd.js
+  - npm install typescript@2.0.7 && npm run build_spec && npm run test_mocha
+  - npm install typescript@latest && npm run build_spec && npm run test_mocha
```

Comment:
> This test is guarding 'current' change to be safe with previous version of TS, while further version comes out and if decided to introduce version specific breaking changes can be taken out.

I'm worrying about that we will need to maintain on 2 different version of compilers if we try to https://github.com/ReactiveX/rxjs/issues/2066, these type based analysis are improved by tsc incrementally, so the difference of compiler causes the situation we cannot compile it on both versions.
So do you means that is is an separate issue which we will face to and we should resolve if we tackle to https://github.com/ReactiveX/rxjs/issues/2066 or others? (If you said so, I agree it.)